data/pokedex.js

```diff
@@ -3756,6 +3756,21 @@ exports.BattlePokedex = {
 		weightkg: 58,
 		color: "White",
 		eggGroups: ["Field"],
+		otherFormes: ["smearglebowtie"],
+	},
+	smearglebowtie: {
+		num: 235,
+		species: "Smeargle-Bowtie",
+		baseSpecies: "Smeargle",
+		forme: "Bowtie",
+		formeLetter: "B",
+		types: ["Normal"],
+		baseStats: {hp: 55, atk: 20, def: 35, spa: 20, spd: 45, spe: 75},
+		abilities: {0: "Own Tempo", 1: "Technician", H: "Moody"},
```

Comment:
Yes, it actually does have all of Smeargle's normal abilities, according to the data Marty mined for me. https://hastebin.com/atujafavol.css